src/olympia/access/acl.py

```diff
@@ -6,10 +6,11 @@ def match_rules(rules, app, action):
     This will match rules found in Group.
     """
     for rule in rules.split(','):
+        if rule == '*:*':
+            return True
         rule_app, rule_action = rule.split(':')
-        if rule_app == '*' or rule_app == app:
-            if rule_action == '*' or rule_action == action or action == '%':
```

Comment:
We still have `amo.permissions.ADMIN` which specifies `%` and that isn't going to work any more.  I'm generally a bit confused about the changes here.

Reply:
`amo.permissions.ADMIN` still works if you have `Admin:%` specifically - i.e. `%` is no longer special. Or if you have `*:*`.

No group has any `Admin:Something` permission currently.

Reply:
If we're dropping `%` as a match wildcard I'd prefer we drop it everywhere - having it in `amo.permissions.ADMIN` implies it means something special.  Maybe change `.ADMIN` to be `Admin:Admin` or even `*:*` to make it obvious.

Reply:
How about changing `ADMIN` to `ALL_ADMIN` and implement https://github.com/mozilla/addons-server/issues/7249 at the same time then ?

Reply:
Deferring until I understand #7249

Reply:
btw, have you had someone check on prod what permission strings are used?  I always planned to reduce all the `Addon:xxx` permissions the staff group had down to `Addon:* for brevity`.  I did it on Marketplace but can't can't remember if I did it - or suggested it be done - on AMO

Reply:
As said above, there isn't any group with `Something:*` or `*:Something `(same with `%`) on prod currently.

Since there is still some discussion going on in #7249... how can we move forward here? I am thinking about dropping all these changes to the wildcards, keep `ADMIN` as before, just find a solution for `is_staff`.

Reply:
Yeah....

Fwiw I can't see %:xxx and *:xxx ever being that useful, but - depending on #7249 - being able to have a user with `Admin:*` meaning "this user has access to all admin tools" without being `*:*` and without listing them all in the group; and being able to test for `Admin:%` for "has the user got any access to admin tools" without enumerating all permissions in the code, both seem useful to have, imo anyway.